### DISCUSSION TITLE
Google reporting mobile usability issues

### DISCUSSION THREAD

**Discussion by billykenny** [viresh-ratnakar/exolve#89] [Q&A]
Google is reporting:
Clickable elements too close together
and
Content wider than screen

for several of my puzzles including [this one:](https://www.hubblecrosswords.com/crossword001.html) 

I have used the "======REPLACE WITH YOUR PUZZLE BELOW======" option and not modified anything from Exolve. 
Any straightforward way to overcome this problem? TIA

**Comment by viresh-ratnakar** [comment]
Sorry, just saw this.

I imagine "clickab;le elements too close together" would in general be not good for a user interface, and some Google check is going by that. However, in the specific case of *crosswords*, clickable cells are naturally next to each other, and i would not worry about such warnings.

**Reply by billykenny** [reply]
Thank you
